recipes/taxoniq/meta.yaml

```
@@ -0,0 +1,50 @@
+{% set name = "taxoniq" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 19fb5d7f85158dc9e80c636635716ce14bb13671abefc9b785257789353f6982
+
+build:
+  number: 0
+  entry_points:
+    - taxoniq=taxoniq.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
```

```suggestion
  build:
    - {{ compiler('c') }}
    host:
```
seems like you need a c compiler

---

Looking at the setup.py it seems to have some cpp extensions as well

---

Thanks, I was just looking into c compiler.